src/main/java/org/mockito/internal/stubbing/answers/ThrowsExceptionClass.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2020 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito.internal.stubbing.answers;
+
+import java.io.Serializable;
+
+import org.mockito.creation.instance.Instantiator;
+import org.mockito.internal.configuration.plugins.Plugins;
+
+public class ThrowsExceptionClass extends AbstractThrowsException implements Serializable {
```

Comment:
Let's name this `ThrowsExceptionForClassType` to make it clear we are passing in the `.class` itself.

Reply:
The change has been done. The class, references and associated variables have been updated.